src/libraries/pkg/baseline/packageIndex.json

```diff
@@ -2550,7 +2550,7 @@
         "4.5.0",
         "4.6.0"
```

Comment:
Should we remove the "System.Json" entry all together or is it still needed?

Reply:
we haven't done it for other dead projects. we keep them for consistency.

Reply:
Keep them in the rare case that we need to build packages in master that might depend on the old versions.  EG: Windows-compat metapackage.